client-app/src/Components/NewItemForm.tsx

```diff
@@ -233,28 +233,6 @@ const NewItemForm: React.FC = () => {
 
                 formDataToSubmit.append('dateDonated', formData.dateDonated);
 
-                // Append each image file directly
-                if (formData.imageUpload.length) {
-                    formData.imageUpload.forEach((image, index) => {
-                        // Convert base64 back to Blob if needed (assuming the backend expects files, not base64 strings)
-                        const byteString = atob(image.split(',')[1]);
-                        const mimeString = image
-                            .split(',')[0]
-                            .split(':')[1]
-                            .split(';')[0];
-                        const ab = new Uint8Array(byteString.length);
-                        for (let i = 0; i < byteString.length; i++) {
-                            ab[i] = byteString.charCodeAt(i);
-                        }
-                        const blob = new Blob([ab], { type: mimeString });
-                        formDataToSubmit.append(
-                            'images',
-                            blob,
-                            `image-${index}.jpg`,
-                        ); // Use a default name
-                    });
-                }
-
                 console.log('formDataToSubmit:', formDataToSubmit);
```

Comment:
@miabu-pashh I think there is a miscommunciation. We want to keep the image upload functionality in `frontend`. I meant to remove the image related functionality from `server` but not from `server`. Please keep the previous and newly added image upload functionality in `frontend `please